docker/Dockerfile

```diff
@@ -1,13 +1,11 @@
 FROM gliderlabs/alpine
 MAINTAINER Weaveworks Inc <help@weave.works>
 WORKDIR /home/weave
-ADD ./app /home/weave/
-ADD ./probe /home/weave/
-ADD ./entrypoint.sh /home/weave/
-ADD ./run-app /etc/service/app/run
-ADD ./run-probe /etc/service/probe/run
-
-RUN echo "http://dl-4.alpinelinux.org/alpine/edge/testing" >>/etc/apk/repositories && apk add --update runit
-
+RUN echo "http://dl-4.alpinelinux.org/alpine/edge/testing" >>/etc/apk/repositories && \
+	apk add --update runit && \
+	rm -rf /var/cache/apk/*
+COPY ./app ./probe ./entrypoint.sh /home/weave/
+COPY ./run-app /etc/service/app/run
+COPY ./run-probe /etc/service/probe/run
```

Comment:
COPY is preferred https://docs.docker.com/articles/dockerfile_best-practices/#add-or-copy